cmd/node-observability-operator/main.go

```diff
@@ -112,11 +113,24 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	// KubeAPI client to be used only in order to get the configmap kubelet-serving-ca
+	// from NS openshift-config-managed.
+	// The clusterWideCli is needed because ConfigMaps are namespaced resources, and in the context
+	// of a namespaced operator, the operator only looks for the namespaced resources in its own namespace
+	// see https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.1/pkg/manager#Options => Namespace
+	clusterWideCli, err := client.New(mgr.GetConfig(), client.Options{
+		Scheme: scheme,
+	})
```

Comment:
As discussed, [MultiNamespaceCacheBuilder](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/cache#MultiNamespacedCacheBuilder) passed to `NewCache` manager option can be used to stay with the single client.